__tests__/date-past-now/date-past-now.test.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-new-func */
+import moment from 'moment';
+import dateExpired from '../../functions/date-past-now/1.0';
+
+describe('Check if a date(time) input is past the current date(time)', () => {
+  test('Check if yesterday is before the current date(time)', async () => {
```

Comment:
Seems like of the logic is reused. Should be D.R.Y.

Reply:
Done